Assets/Scripts/Items/SpeedUpItem.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpeedUpItem : MonoBehaviour
+{
+    public float speedUpAmount;
+
+    private bool isSpawned = false;
+    private int row = -1;
+    private int col = -1;
+
+    void Update() {
+        if (isSpawned && !Level.GetSingleton().HasLilypadAt(row, col)) {
```

Comment:
Since `Update()` is called many times a second, maybe you want to cache the level singleton at `Start()`?